src/stylesheets/components/_rating.scss

```diff
@@ -0,0 +1,36 @@
+.rating {
+  margin-top: rem(45px);
+
+  &__list {
+    width: fit-content;
+    margin: 0 auto;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__emoji {
+    display: inline;
+
+    font-size: map-get($font-sizes, 'xxl');
+    line-height: rem(34px);
+    letter-spacing: rem(16px);
+
+    cursor: pointer;
+    opacity: 0.5;
+
+    transition: color 200ms;
+  }
+
+  &__list:hover > &__emoji {
+    opacity: 1;
+  }
+
+  &__list > &__emoji:hover ~ &__emoji {
+    opacity: 0.5;
+  }
+
+  &__emoji--selected {
+    opacity: 1;
+  }
```

Comment:
@rafayet-monon Should we group like this below?

```suggestion
  &__emoji {
    display: inline;

    font-size: map-get($font-sizes, 'xxl');
    line-height: rem(34px);
    letter-spacing: rem(16px);

    cursor: pointer;

    transition: color 200ms;
  }
  
  &__emoji,
  &__list > &__emoji:hover ~ &__emoji {
    opacity: 0.5;
  }

  &__list:hover > &__emoji,
  &__emoji--selected {
    opacity: 1;
  }
```

Reply:
Fixed in [e2af2e5](https://github.com/rafayet-monon/surveyor/pull/20/commits/e2af2e51e5cc86f8a19cf1cc801c6bbcaa66f0d2)